CMSgov/design-system
# [NO-TICKET] Add link variables to link page

## Summary

- Add link variables to link page

## How to test

1. Run `yarn start` and visit the link page and see that the variables are listed under the styles section

## Checklist




### If this is a change to code:


### If this is a change to documentation:
